utils/download_test_db.py

```diff
@@ -121,31 +121,88 @@ def download_dataset(dataset):
             # Just a newline.
             print()
 
+def download_caffe2_models ():
+    models = ['densenet121',
+              'inception_v1', 
+              'inception_v2',
+              'lenet_mnist',
+              'resnet50',
+              'shufflenet',
+              'squeezenet',
+              'vgg19', 
+              'zfnet512',
+              'bvlc_alexnet',
+              'en2gr',
+              'quant_resnet50']
+    for modelname in models:
+        for filename in ['predict_net.pbtxt', 'predict_net.pb', 'init_net.pb']:
+            os.system ('wget -nc http://fb-glow-assets.s3.amazonaws.com/models/{}/{} -P  {}'.format(modelname, filename, modelname))
+    
+    for filename in ['dst_dictionary.txt', 'src_dictionary.txt']:
+        os.system ('wget -nc http://fb-glow-assets.s3.amazonaws.com/models/en2gr/{} -P en2gr'.format(filename))
+    return
+
+def download_onnx_models ():
+    for modelname in ['resnet50', 'vgg19', 'squeezenet', 'zfnet512', 'densenet121', 'shufflenet']:
+        os.system ('wget -nc https://s3.amazonaws.com/download.onnx/models/opset_6/{}.tar.gz'.format(modelname))
+        os.system ('tar -xzvf {}.tar.gz'.format(modelname))
+
+    for modelname in ['inception_v1', 'inception_v2', 'bvlc_alexnet']:
+        os.system ('wget -nc https://s3.amazonaws.com/download.onnx/models/opset_8/{}.tar.gz'.format(modelname))
+        os.system ('tar -xzvf {}.tar.gz'.format(modelname))
+
+    for modelname in ['lenet_mnist']:
+        os.system ('wget -nc http://fb-glow-assets.s3.amazonaws.com/models/{}.tar.gz'.format(modelname))
+        os.system ('tar -xzvf {}.tar.gz'.format(modelname))
+
+    for modelname in ['googlenet_v1_slim', 'googlenet_v4_slim', 'resnet50_tf']:
+        os.system ('mkdir {}'.format(modelname))    
+        os.system ('wget -nc -P {} http://fb-glow-assets.s3.amazonaws.com/models/{}.onnx'.format(modelname, modelname))
+
+    os.system ('wget -nc https://onnxzoo.blob.core.windows.net/models/opset_8/emotion_ferplus/emotion_ferplus.tar.gz')
+    os.system ('tar -xzvf emotion_ferplus.tar.gz emotion_ferplus/model.onnx')
+
+    os.system ('wget -nc https://s3.amazonaws.com/download.onnx/models/opset_8/bvlc_reference_rcnn_ilsvrc13.tar.gz')
+    os.system ('tar -xzvf bvlc_reference_rcnn_ilsvrc13.tar.gz bvlc_reference_rcnn_ilsvrc13/model.onnx')
+    return
 
 def parse():
     parser = argparse.ArgumentParser(description='Download datasets for Glow')
     parser.add_argument('-d', '--datasets', nargs='+', choices=DATASET_NAMES)
-    parser.add_argument('-a', '--all', action='store_true')
+    parser.add_argument('-ad', '--allDatasets', action='store_true')
```

Comment:
Nit: I don't think camel case is the convention for command-line flags. Consider using `--all-datasets`, `--caffe2-models`, and `--onnx-models` instead.

Reply:
agree,  let's match the style of our C++ utils